### ISSUE TITLE
show image thumbnails in the list

### ISSUE THREAD

**Issue by unfa** [polypmer/fissure-android#6]
Right now the user can only preview images ony by one, by touching them. Would it be a good thing to show all the image thumbnails in the list?

**Comment by fenimore** [comment]
Yes it would be nice, but I worry it might slow down the application,

**Comment by fenimore** [comment]
Also, now that one is able to add multiple images at once, I think this is even less urgent.
